src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public enum Colors {
+
```

Comment:
I see that in some places you add extra new line at the beginning of block whereas in other you don't. 
Please see that according to `Common mistakes` doc, you shouldn't add any new lines here: https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced